packages/adapter-node/src/polka-handler.js

```diff
@@ -38,7 +38,7 @@ export function createServer({ render }) {
 		  })
 		: noop_handler;
 
-	const server = polka().use(
+	const polka_instance = polka().use(
```

Comment:
I think I might expect at least `compression` to be added in the reference server implementation rather than here. That way if someone wants to change it they can.

Similarly, I might export `prerenderedHandler` and `assetsHandler` and add them to the reference implementation rather than adding them automatically here in cause someone wants to provide a different implementation